integration-cli/docker_api_containers_test.go

```diff
@@ -138,11 +138,7 @@ func TestContainerApiStartVolumeBinds(t *testing.T) {
 		t.Fatal(err)
 	}
 
-	bindPath, err := ioutil.TempDir(os.TempDir(), "test")
-	if err != nil {
-		t.Fatal(err)
-	}
-
+	bindPath := path.Join("/tmp", fmt.Sprintf("test.%s", makeRandomString(10)))
```

Comment:
If I understood this PR correctly: because of https://github.com/docker/docker/pull/10794#issue-57684837; filepath would attempt to create a Windows-style path (`C:\temp`) on the daemon, but the daemon is running on linux